scripts/abundance-dist-single.py

```diff
@@ -1,7 +1,7 @@
 #! /usr/bin/env python
 # This file is part of khmer, https://github.com/dib-lab/khmer/, and is
-# Copyright (C) 2010-2015, Michigan State University.
-# Copyright (C) 2015, The Regents of the University of California.
+# Copyright (C) 2010-2016, Michigan State University.
```

Comment:
Thanks for updating the headers, but unless you work for MSU this one doesn't need updating as no one in dib-lab works for MSU anymore :-)